modules/sso/config/launch/setup/73/added/launch/openshift-x509.sh

```diff
@@ -90,10 +94,10 @@ function autogenerate_keystores() {
     fi
 
     # Import existing system CA certificates into the newly generated truststore
-    local SYSTEM_CACERTS=$(readlink -e $(dirname $(readlink -e $(which keytool)))"/../lib/security/cacerts")
-    if keytool -v -list -keystore "${SYSTEM_CACERTS}" -storepass "changeit" > /dev/null; then
+    local SYSTEM_CACERTS=$(readlink -e $(dirname $(readlink -e $(which "$KEYTOOL")))"/../lib/security/cacerts")
```

Comment:
@hmlnarik 

For OpenJ9 (tried on `adoptopenjdk-8-openj9-8u232_b09.openj9_0.17.0-1.x86_64`) the path to `cacerts` file seems to be:
```
# rpm -ql adoptopenjdk-8-openj9 | grep cacerts
/usr/lib/jvm/adoptopenjdk-8-openj9/jre/lib/security/cacerts
```
Notice the more `/jre`added to the path. If `keytool` from `/usr/lib/jvm/adoptopenjdk-8-openj9/jre/bin/keytool` (the JRE one` would be used, the above `SYSTEM_CACERTS` path would work. But if the JDK OpenJ9 keytool would be used (`/usr/lib/jvm/adoptopenjdk-8-openj9/bin/keytool`) the `SYSTEM_CACERTS` path argument `"/../lib/security/cacerts"` should be replaced with `"/../jre/lib/security/cacerts"` instead IMHO.

Should both OpenJ9 paths of `keytool` tool be supported? If so, the `SYSTEM_CACERTS` should append a slightly different suffix in either of those cases in that case. WDYT?

Reply:
I'd say we only use the `jre` one then.

Reply:
@hmlnarik Ok, so in that case could you update the PR so it contains `jre` in the alternative path to the `keytool` binary? Something along the lines (but untested):

```
 if [ ! -x "$KEYTOOL" -a -x "$JAVA_HOME/jre/bin/keytool" ]; then
        KEYTOOL="$JAVA_HOME/jre/bin/keytool"
```

AFAICT / AFAICS this should be sufficient, the `SYSTEM_CACERTS` to be set correctly also in the alternative case (but would be good to test & verify it of course)

Thanks!